galaxyxml/tool/parameters/__init__.py

```diff
@@ -363,6 +393,7 @@ class When(InputParameter):
     name = "when"
 
     def __init__(self, value):
+        self.value = value
```

Comment:
Note I need this in the cli generation of conditionals. But alternatively I could use something like `WHEN.node.attribs["value"]`. Any preferences?

Reply:
Or we implement an `__getattr__` in `XMLParam` that gets the data from `node.attribs` in general .. I somehow like that Idea :)

Reply:
`__getattr__` definitely sounds better to me, too. :)